### PR TITLE
feat: implement notion restart-all command

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: connectors/src/admin/cli.ts [hunk 1]
```diff
@@ -74,6 +75,66 @@ const connectors = async (command: string, args: parseArgs.ParsedArgs) => {
   }
 };
 
+const notion = async (command: string) => {
+  switch (command) {
+    case "restart-all": {
+      const queue = new PQueue({ concurrency: 10 });
+      const promises: Promise<void>[] = [];
+      const connectors = await Connector.findAll({
```
Comment (fontanierh) [inline connectors/src/admin/cli.ts:83]: obviously this won't work forever @spolu

Comment (spolu) [reply connectors/src/admin/cli.ts:83]: :+1:

**Review by spolu — DISMISSED** [review]

**Comment by fontanierh** [comment]
I don't understand the CI 😢 npm run build is working locally

**Comment by spolu** [comment]
@fontanierh surprising there is no package.json / package-lock.json update maybe related?

**Comment by fontanierh** [comment]
Yep, indeed, fixed thanks @spolu

**Review by spolu — APPROVED** [review]
\o/

**Pull request merged**